### DISCUSSION TITLE
Who feels responsible fro this repository? Is it still maintained?

### DISCUSSION THREAD

**Discussion by laeubi** [xerial/sqlite-jdbc#739] [Q&A] [answered]
I see a lot of (old) PRs hanging around and some even haven't got any feed back since now, so I wonder if this repository is actively maintained, or if there are other channels beside github (discussions) to get in touch with the developers to negotiate about changes and releases of this library?

**Comment by re-thc** [comment]
Is there a fork or would anyone be interested in 1?

**Comment by gotson** [comment, marked as answer]
Hello !
xerial is very busy and cannot spend any time on this project at the moment, but i have good news ! I contacted him, and i just got added as a contributor to the repository.

I will start going through the PRs and issues to get things moving.

**Comment by xerial** [comment]
Thank @gotson for stepping up as a maintainer of the project! 

My attention to maintain this project has not been evenly placed, simply because I've been maintaining this project for 10+ years (!!!), and also minimum features I need in sqlite-jdbc have already been implemented. 

It's good if the project is driven by people who actually need new features and fixes. In this sense, @gotson is the right person at this moment. Once the release process is automated, the maintenance work will be much easier.
